docs/sources/operators-guide/configure/configure-object-storage-backend.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Configure Grafana Mimir object storage backend"
+menuTitle: "Configure object storage"
+description: "Learn how to configure Grafana Mimir to use different object storage backend implementations."
+weight: 70
+---
+
+# Configure Grafana Mimir object storage backend
+
+Grafana Mimir can use different object storage services to persist blocks containing the metrics data, as well as recording rules and alertmanager state.
+The supported backends are:
+
+- [Amazon S3](https://aws.amazon.com/s3/) (and compatible implementations like [MinIO](https://min.io/))
+- [Google Cloud Storage](https://cloud.google.com/storage)
+- [Azure Blob Storage](https://azure.microsoft.com/es-es/services/storage/blobs/)
+- [Swift (OpenStack Object Storage)](https://wiki.openstack.org/wiki/Swift)
+
+Additionally, a file system emulated [`filesystem`]({{< relref "../configure/reference-configuration-parameters/index.md#filesystem_storage_backend" >}}) object storage implementation can be also used for testing purposes (it's not recommended for production workloads).
+
+[Ruler and alertmanager support a `local` implementation]({{< relref "../architecture/components/ruler/index.md#local-storage" >}}), which is similar to `filesystem` in the way that it uses the local file system, but it is a read-only data source and can be used to provision state into those components.
+
+## Common configuration
+
+The [common configuration]({{< relref "about-configurations.md#common-configurations" >}}) can be used to avoid repetition by filling the [`common`]({{< relref "../configure/reference-configuration-parameters/index.md#common" >}}) configuration block or by providing the `-common.storage.*` CLI flags.
```

Comment:
```suggestion
To avoid repetition, you can use the [common configuration]({{< relref "about-configurations.md#common-configurations" >}}) and fill the [`common`]({{< relref "../configure/reference-configuration-parameters/index.md#common" >}}) configuration block or by providing the `-common.storage.*` CLI flags.
```
Use semantic line breaks.

Reply:
Reworded to change passive voice to active voice.